src/main/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysis.java

```diff
@@ -108,6 +105,7 @@ SecurityAnalysisResult runSync(SecurityAnalysisParameters securityAnalysisParame
 
         LoadFlowParameters lfParameters = securityAnalysisParameters.getLoadFlowParameters();
         OpenLoadFlowParameters lfParametersExt = OpenLoadFlowProvider.getParametersExt(securityAnalysisParameters.getLoadFlowParameters());
+        lfParametersExt.setThrowsExceptionInCaseOfSlackDistributionFailure(false); //FIXME
```

Comment:
I have added this line in case of a special case where a contingency leads to the loss of a great part of the generation. In that case, the slack distribution could failed. But, I am not sure that we have to keep this parameter for base case.